apps/src/templates/instructions/utils.js

```diff
@@ -35,6 +35,40 @@ export function scrollTo(element, scrollTop, animate=400) {
   }
 }
 
+/**
+ * Converts any inline XML in the container element into embedded
+ * readonly BlockSpaces
+ * @param {Element} container The element in which to search for XML
+ */
+export function convertXmlToBlockly(container) {
+  const xmls = container.getElementsByTagName('xml');
+  Array.prototype.forEach.call(xmls, function (xml) {
+    // create a container and insert the blockspace into it
+    const container = xml.parentNode.insertBefore(document.createElement('div'), xml);
```

Comment:
Unfortunately no; components have to be mounted in the `render` method, and the xml isn't created (from the `dangerouslySetInnerHtml` param) until the parent component is mounted.